sympy/integrals/__init__.py

```diff
@@ -9,6 +9,7 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
```

Comment:
Aaron confirmed that it's the right fix, and Travis confirms "no problem" on my own repo, so this should be good now.